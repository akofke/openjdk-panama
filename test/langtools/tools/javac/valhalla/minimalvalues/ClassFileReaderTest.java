/*
 * @test /nodynamiccopyright/
 * @summary Verify that the class reader flags Value capable classes appropriately.
 * @modules jdk.incubator.mvt
 * @compile Point.java
 * @compile/fail/ref=ClassFileReaderTest.out --should-stop:ifError=PARSE -XDrawDiagnostics -Werror -Xlint:values  ClassFileReaderTest.java
 */

public class ClassFileReaderTest {
    Point point = null;
}
