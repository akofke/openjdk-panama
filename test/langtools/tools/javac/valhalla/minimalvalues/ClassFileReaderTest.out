- compiler.warn.incubating.modules: jdk.incubator.mvt
- compiler.err.warnings.and.werror
ClassFileReaderTest.java:10:19: compiler.warn.prob.found.req: (compiler.misc.inconvertible.types: compiler.misc.type.null, Point)
1 error
2 warnings
